models/src/main/scala/coop/rchain/casper/PrettyPrinter.scala

```diff
@@ -38,12 +39,14 @@ object PrettyPrinter {
     }
 
   def buildString(d: ProcessedDeploy): String =
-    s"User: ${buildStringNoLimit(d.deploy.deployer)}, Cost: ${d.cost.toString} " +
+    s"User: ${buildStringNoLimit(d.deploy.pk.bytes)}, Cost: ${d.cost.toString} " +
       s"${buildString(d.deploy)}"
 
   def buildString(b: ByteString): String =
     limit(Base16.encode(b.toByteArray), 10)
 
+  def buildString(sd: Signed[DeployData]): String =
+    s"${buildString(sd.data)}, SigAlgorithm: ${sd.sigAlgorithm.name}, Sig: ${buildString(sd.sig)}"
```

Comment:
@zsluedem signature is now added at the end. Also I put signature algorithm in front because it's much shorter.